README.md

```diff
@@ -68,11 +37,11 @@ uv sync
 cp .env.sample .env
 # Edit .env with your configuration
 
-# Run database migrations  
-uv run python -m migrations.runner
+# Run database migrations
+uv run python -m tiger_agent.migrations.runner
 
-# Start the agent
-uv run python -m app.main
+# Start the harness
+uv run python -m tiger_agent.main
```

Comment:
This probably isn't correct since don't we require the consumer to write a script that triggers the harness?